tests/test_customer_client.py

```diff
@@ -40,6 +40,11 @@ def create_customer():
             document_locale="fr",
             provider_payment_methods=["card", "sepa_debit"],
         ),
+        shipping_address=Address(
+            city='Paris',
+            zipcode='123',
+            country='FR'
+,       ),
```

Comment:
would you consider include all the attributes of the address in the tests? address_line 1 & 2, and state